configs/builds.json

```diff
@@ -98,7 +98,7 @@
 		},
 		{
 			"target": "esp32",
-			"features":["qio_ram"],
+			"features":["qio_ram","insights"],
```

Comment:
so this would only work on ESP32?

Reply:
Yes, I have only added to esp32 target, since It is the only target I have tested, but I guess there won't be a problem to add it for the other SoC series. Should I create another PR for them?

Reply:
if you add it to the others, we can accommodate the change in the next 2.0.6 release. Plus we would not need to limit the example to ESP32 only. Keep in mind that ESP32 support is already in Arduino master, so make sure you get it to worm in Arduino as well (there are some errors)

Reply:
Until the last days (https://github.com/espressif/esp32-arduino-lib-builder/commit/296fda565d5ea353eabb9262b054ff5e96db1bf7#diff-7dc73c61faafeb3cfa84a00d17773425a77c0140f34d8248f8085b82c4dc97a3), the bootloader was created for every flash memory type and so on but now only the elf files are copied to arduino-esp32. Do you know where could I find now the bootloader used while compiling using PlatformIO?

Reply:
Fix for platformio is coming. Necessary changes are already merged in Arduino master, so I guess you will have the full support soon on PIO side as well.

Reply:
I see, I will check what are those changes regarding the bootloader generation, this ESP-Insights feature requires also running the esptool with _--elf-sha256-offset_ flag and I made another PR to https://github.com/platformio/platform-espressif32/pull/973 but it seems that it is not yet so clear if this will affect too the bootloader flash flags...
Thank you @me-no-dev!